domain/src/main/java/org/oppia/android/domain/feedbackreporting/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+"""
+Domain services & definitions corresponding to user feedback reporting.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "feedback_reporting_module",
```

Comment:
Please call this prod_module for consistency.

Reply:
done